sfge/src/main/java/com/salesforce/graph/ops/MethodUtil.java

```diff
@@ -257,6 +260,21 @@ public static List<MethodVertex> getGlobalMethods(
                                         __.not(__.has(Schema.IS_STANDARD, true)))));
     }
 
+    public static List<MethodVertex> getInboundEmailHandlerMethods(
+            GraphTraversalSource g, List<String> targetFiles) {
+        return SFVertexFactory.loadVertices(
+                g,
+                // Get any target class that implements the email handler interface.
+                TraversalUtil.traverseImplementationsOf(g, targetFiles, INBOUND_EMAIL_HANDLER)
+                        // Get every implementation of the handle email method.
+                        .out(Schema.CHILD)
+                        .where(H.has(NodeType.METHOD, Schema.NAME, HANDLE_INBOUND_EMAIL))
+                        // Filter the results by return type and arity to limit the possibility of
+                        // getting unnecessary results.
+                        .where(H.has(NodeType.METHOD, Schema.RETURN_TYPE, INBOUND_EMAIL_RESULT))
+                        .has(Schema.ARITY, 2));
```

Comment:
At this time, we're only checking the arity, instead of checking the types of arguments. We could add that in the future, but that feels like an excessive level of complication unless we know we'll need it. (He said, having already overcomplicated the PR...)